src/flynt/static_join/candidates.py

```diff
@@ -0,0 +1,35 @@
+import ast
+from typing import List
+
+from flynt import state
+from flynt.ast_chunk import AstChunk
+from flynt.static_join.utils import (
+    get_static_join_bits,
+)
+
+
+class JoinHound(ast.NodeVisitor):
+    def __init__(self):
+        super().__init__()
+        self.victims: List[AstChunk] = []
+
+    def visit_Call(self, node: ast.Call):
+        """
+        Finds all nodes that are joins with a static string literal
+        as the joiner and a static iterable as the joinee.
+        """
+        if get_static_join_bits(node) is not None:
+            self.victims.append(AstChunk(node))
+        else:
+            self.generic_visit(node)
+
+
+def join_candidates(code: str):
```

Comment:
what is the advantage of making this a generator? can we return a collection?

Reply:
We can, sure. I just basically copied this from https://github.com/ikamensh/flynt/blob/071dfb012adca6c13a311bc2f393b8a0907e9cc5/src/flynt/string_concat/candidates.py#L37-L45 :)